app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/TemporaryAdmissionMethodOfDisposal.scala

```diff
@@ -48,4 +48,18 @@ object TemporaryAdmissionMethodOfDisposal extends EnumerationFormat[TemporaryAdm
       Other,
       MultipleDisposalMethodsWereUsed
     )
+
+  val orderedValues: List[TemporaryAdmissionMethodOfDisposal] = List[TemporaryAdmissionMethodOfDisposal](
```

Comment:
I think this should be decided on the view level, like https://github.com/hmrc/cds-reimbursement-claim-frontend/blob/f673a702b0fa5aeabe029ce85497dd2bdf77b5e2/app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/securities/choose_reason_for_security.scala.html#L42-L54

Reply:
Okay, but the order of the descriptions in the prototype is not alphabetical...

Reply:
Jojest confirmed ordering is mandatory. I've moved the ordering to the presentation logic in the latest commit